### PR TITLE
Update YouCube

### PR DESCRIPTION
**Pull request by Commandcracker**

### CODE REVIEW

File: basalt.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+-- TODO: [Github releases provider](https://github.com/unicornpkg/wing/issues/7)
+-- https://github.com/Pyroxenium/Basalt/releases/download/v1.6.3/basalt.lua
+
+local package = {}
+package.name = "basalt"
+package.version = "1.6.3"
+package.instdat = {}
+package.instdat.repo_owner = "pyroxenium"
+package.instdat.repo_name = "basalt"
```
Comment (tomodachi94) [inline basalt.lua]: For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_name = "Basalt"
```

File: basalt.lua [hunk 2]
```diff
@@ -0,0 +1,18 @@
+-- https://github.com/Pyroxenium/Basalt/
+
+-- TODO: [Github releases provider](https://github.com/unicornpkg/wing/issues/7)
+-- https://github.com/Pyroxenium/Basalt/releases/download/v1.6.3/basalt.lua
+
+local package = {}
+package.name = "basalt"
+package.version = "1.6.3"
+package.instdat = {}
+package.instdat.repo_owner = "pyroxenium"
```
Comment (tomodachi94) [inline basalt.lua]: For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_owner = "Pyroxenium"
```

File: basalt.lua [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+package.instdat.repo_name = "basalt"
+package.instdat.repo_ref = "1.6.3"
```
Comment (tomodachi94) [inline basalt.lua]: This repository ref is inaccurate. It should point to a valid Git tag.

See [*The Git Book: 10.3 Git References*](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
```suggestion
package.instdat.repo_ref = "v1.6.3"
```

File: youcube.lua [hunk 4]
```diff
@@ -0,0 +1,18 @@
+-- https://github.com/Commandcracker/YouCube
```
Comment (tomodachi94) [inline youcube.lua:1]: Are there versions for this project?

File: youcube.lua [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+local package = {}
+package.name = "youcube"
+package.instdat = {}
+package.instdat.repo_owner = "Commandcracker"
+package.instdat.repo_name = "youcube"
```
Comment (tomodachi94) [inline youcube.lua]: For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_name = "YouCube"
```

File: youcubeapi.lua [hunk 6]
```diff
@@ -0,0 +1,14 @@
+-- https://github.com/Commandcracker/YouCube
+
+local package = {}
+package.name = "youcubeapi"
+package.instdat = {}
+package.instdat.repo_owner = "commandcracker"
```
Comment (tomodachi94) [inline youcubeapi.lua]: For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_owner = "Commandcracker"
```

File: youcubeapi.lua [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+package.instdat.repo_name = "youcube"
```
Comment (tomodachi94) [inline youcubeapi.lua]: For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_name = "YouCube"
```

**Review by tomodachi94 — CHANGES_REQUESTED** [review]
See my requested changes.

**Review by Commandcracker — COMMENTED** [review]
i'll stick to the capitalization :+1:

**Review by tomodachi94 — APPROVED** [review]
Lgtm.

**Pull request merged**